### ISSUE TITLE
Add spell check support

### ISSUE THREAD

**Issue by jessegrosjean** [FoldingText/foldingtext-for-atom#43]

**Comment by betahat** [comment]
Is this possible by including the correct scope in the grammars list to use Atom's spell check? I'm not sure what the right grammar for a ftml file would be, though.

**Comment by jessegrosjean** [comment]
I don't think this is possible. FoldingText for Atom doesn't use the standard Atom text buffer, it's build using a custom HTML/DOM view, so can't reuse any of that functionality.
